pkg/proc/proc.go

```diff
@@ -140,6 +120,7 @@ func (dbp *Process) Detach(kill bool) (err error) {
 			err = killProcess(dbp.pid)
 		}
 	})
+	dbp.Close()
```

Comment:
Seems odd to call close on `Process` -- makes me think of not embedding `BinaryInfo`, but could be convinced otherwise.